golem_messages/message/base.py

```diff
@@ -0,0 +1,382 @@
+import calendar
+import datetime
+import enum
+import hashlib
+import logging
+import struct
+import time
+
+import golem_messages
+
+from golem_messages import datastructures
+from golem_messages import exceptions
+from golem_messages import serializer
+from golem_messages import settings
+
+logger = logging.getLogger('golem.network.transport.message')

```

Comment:
no default `__name__` for all loggers?

Reply:
It was left like this for backwards compatibility (before moving `golem.network.transport.message` to `golem_messages.message` and then `golem_messages.message.base`)